### ISSUE TITLE
`Unresolved reference` in `extensions-compose-jetbrains` of version 1.0.0-alpha-04

### ISSUE THREAD

**Issue by YektaDev** [arkivanov/Decompose#190]
I used to have a dependency on `com.arkivanov.decompose.extensions.compose.jetbrains.stack.Children` in a source file located in a `commonMain` directory and I used it with no problems; but when I bumped the library version from `0.8.0` to `1.0.0-alpha-04`, I got `Unresolved reference: Children` error, which is not true since I can still compile and run the app for all my targets (android & JVM).
I checked to see how deep can the auto-complete recognize the import package and it does not show the `.extentions` part which is actually related to this dependency:
```kotlin
kotlin.sourceSets {
  commonMain {
    dependencies {
      // ...
      implementation("com.arkivanov.decompose:extensions-compose-jetbrains:1.0.0-alpha-04")
    }
    // ...
  }
  // ...
}
```
I tried:
- Doing a clean build (multiple times)
- Invalidating caches & restarting (+ deleting all caches of GRADLE_HOME)
- Switching from latest AS Canary to the latest AS RC
- Downgrading AGP from `7.4.0-alpha09` to `7.3.0-rc01`

... And it still shows the error while I'm able to build and run with no issues! Switching only `extensions-compose-jetbrains` back to `0.8.0` seems to fix the error; so I found out with all I've done, the problem is on the side of `extensions-compose-jetbrains`.

**Comment by malliaridis** [comment]
I faced this issue too. I switched from Android Studio to Intellij IDEA and it worked fine again, the reason is unknown tho.

**Comment by arkivanov** [comment]
@YektaDev If you are using Android Studio, there are known issues of the IDe not being able to resolve multiplatform dependencies. This happens with Kotlin 1.7.x, Decompose `1.0.0-alpha-04` uses Kotlin 1.7.0, and Decompose `0.8.0` uses Kotlin `1.6.10`.

I remember people saying that the issue should be fixed in the latest Android Studio Electric Eel.

**Comment by YektaDev** [comment]
@arkivanov Thanks for mentioning that! Actually, I first encountered this issue with AS Electric Eel that used AGP `7.4.0-alpha09` *(which was the latest version until about two days ago)*, but then the newer version came out that used `7.4.0-alpha10` which had [another bug](https://issuetracker.google.com/issues/243955195) with KMP that led to not being able to even sync the project! So I had to switch to AS RC. I guess I just use the `0.8.0` until the AS bug is fixed.
Should I close this issue or leave it open?
